src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.hpp

```diff
@@ -38,6 +39,20 @@ class FieldSpecificationBase;
 class FiniteElementBase;
 class DomainPartition;
 
+/**
+ * @enum TimeIntegrationOption
+ *
+ * The options for time integration
+ */
+enum class TimeIntegrationOption : int
+{
+  QuasiStatic,    //!< QuasiStatic
+  ImplicitDynamic,//!< ImplicitDynamic
+  ExplicitDynamic //!< ExplicitDynamic
+};
+
+ENUM_STRINGS( TimeIntegrationOption, "QuasiStatic", "ImplicitDynamic", "ExplicitDynamic" )
```

Comment:
Now that enum declaration is pretty short, I moved it back into the solver header. Ideally it should even be moved into the `SolidMechanicsLagrangianFEM` class scope. I may do this later.